AppStoreExample/AppStoreExample/ViewModels/AppSyncViewModel.swift

```diff
@@ -8,43 +8,51 @@
 
 import Foundation
 
+enum AppSyncActions {
+    case idle
+    case dataWasFetched(error: Error?)
```

Comment:
I don't think we want to pass an Error object but just the error message... since this is oriented to be a model for a view.